pulp.spec

```diff
@@ -476,6 +476,8 @@ KEY_PATH="$KEY_DIR/rsa.key"
 KEY_PATH_PUB="$KEY_DIR/rsa_pub.key"
 if [ ! -f $KEY_PATH ]
 then
+  # Ensure the key generated is only readable by the owner.
+  umask 077
```

Comment:
The way I thought scriptlets worked was it set up a new shell for each one... But I should be up front and say I don't really know much about the internals of RPM with respect to scriptlets. 

Given that I don't know with certainty how they work, I think it would indeed be best to stash the umask and change it back later. That way I don't have to spend lots of time reading man pages just so I can be "safe" at the expense of clarity.